Mixpanel/Mixpanel.m

```diff
@@ -1144,6 +1173,25 @@ - (NSString *)currentRadio
     return @"";
 }
 
+- (void)optOutTracking
+{
+    [[NSUserDefaults standardUserDefaults] setBool:YES forKey:kMPOptOutFlag];
+    self.eventsQueue = [NSMutableArray array];
+    self.peopleQueue = [NSMutableArray array];
+    [self archiveEvents];
+    [self archivePeople];
```

Comment:
i think anything that's currently on the dispatch queues will still be added to `eventsQueue` and `peopleQueue` and archived. though they won't be sent because of the change you made to `MPNetwork`, if the user opts back in to tracking, those events will be sent, which i don't think we want